lib/iris/common/metadata.py

```diff
@@ -1198,7 +1198,7 @@ def _check(item):
 
 class DimCoordMetadata(CoordMetadata):
     """
-    Metadata container for a :class:`~iris.coords.DimCoord"
+    Metadata container for a :class:`~iris.coords.DimCoord`
```

Comment:
@tkknight This was just wrong, and causing a documentation warning during `sphinx` build time.